sklearn/preprocessing/label.py

```diff
@@ -148,8 +149,9 @@ def inverse_transform(self, y):
         check_is_fitted(self, 'classes_')
 
         diff = np.setdiff1d(y, np.arange(len(self.classes_)))
-        if diff:
-            raise ValueError("y contains new labels: %s" % str(diff))
+        if len(diff):
```

Comment:
This len is the fix, right?

Reply:
Correct - that's all that was needed to produce a sensible error :smile: